tests/test_fields.py

```diff
@@ -740,6 +744,18 @@ def test_null_bytes(self):
                 'Null characters are not allowed.'
             ]
 
+    def test_iterable_validators(self):
```

Comment:
I might also add a comment/docstring indicating the purpose of the test. e.g., 

```python
# Ensure `validators` is compatible with both lists and tuples.
```

Reply:
done